src/test/java/org/threadly/litesockets/tcp/TCPTests.java

```diff
@@ -117,8 +117,9 @@ public void setClientOptions() throws IOException, InterruptedException {
     }
     
     client.close();
-    
+    Thread.sleep(100);
```

Comment:
yeah, not sure a good way to handle it, I am open to suggestions.  Basically the close is now async and we cant do the checks below till it happens.